ivy/utils/exceptions.py

```diff
@@ -390,6 +397,25 @@ def _handle_exceptions(*args, **kwargs):
             raise ivy.utils.exceptions.IvyBackendException(
                 fn.__name__, str(e), include_backend=True
             )
+        except InplaceUpdateException as e:
+            _configure_stack_trace(e.__traceback__)
+            raise ivy.utils.exceptions.InplaceUpdateException(
+                fn.__name__, str(e), include_backend=True
+            )
 
     _handle_exceptions.handle_exceptions = True
     return _handle_exceptions
+
+
+def handle_inplace_mode():
```

Comment:
let's probably also make this private as users shouldn't really need to call this function